docs/source/getting-started.rst

```diff
@@ -1,65 +1,30 @@
-Getting started with OAuthenticator
-===================================
-
-TODO:
-
-
-Steps
+Get started
+===========
 
+The general steps to take when using OAuthenticator:
 
 1. pick your provider
 2. register with your provider
 3. configure JupyterHub (if supported, pick oauthenticator class, otherwise see :doc:`writing-an-oauthenticator`).
 4. common configuration (client_id, client_secret, callback_url, whitelist, etc.)
 5. specific configuration for your provider
 
-OAuth + JupyterHub Authenticator = OAuthenticator
-
-OAuthenticator currently supports the following authentication services:
-
--  `Auth0 <oauthenticator/auth0.py>`__
--  `Azure AD <#azure-ad-setup>`__
--  `Bitbucket <oauthenticator/bitbucket.py>`__
--  `CILogon <oauthenticator/cilogon.py>`__
--  `GitHub <#github-setup>`__
--  `GitLab <#gitlab-setup>`__
--  `Globus <#globus-setup>`__
--  `Google <#google-setup>`__
--  `MediaWiki <oauthenticator/mediawiki.py>`__
--  `Moodle <#moodle-setup>`__
--  `Okpy <#okpyauthenticator>`__
--  `OpenShift <#openshift-setup>`__
-
-A `generic implementation <oauthenticator.generic.GenericOAuthenticator>`, which you can
-use with any provider, is also available.
-
-Examples
---------
-
-For an example docker image using OAuthenticator, see the
-`examples <examples>`__ directory.
+OAuthenticator currently supports the following **authentication services**:
```

Comment:
Here "authentication services" are used, hmmm.. Since "provider" (which is referring to "identity provider") has been used before, I like to use that here as well.